File: SDKs/googleapis.com/mybusinessaccountmanagement/v1/java/lib/src/main/java/openapisdk/models/shared/TargetLocation.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * TargetLocation
 * Represents a target location for a pending invitation.
**/
public class TargetLocation {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("address")
    public String address;
    public TargetLocation withAddress(String address) {
        this.address = address;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("locationName")
    public String locationName;
    public TargetLocation withLocationName(String locationName) {
        this.locationName = locationName;
        return this;
    }
}

File: SDKs/googleapis.com/mybusinessaccountmanagement/v1/python/src/sdk/models/shared/targetlocation.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class TargetLocation:
    r"""TargetLocation
    Represents a target location for a pending invitation.
    """
    
    address: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('address') }})
    location_name: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('locationName') }})
    
